add changelog for kong-enterprise-k8s 2.0.2.0 release (#1891)
Kong/docs.konghq.com: app/enterprise/k8s-changelog.md

title: Kong Enterprise k8s Changelog
---

## 2.0.2.0

> Released on 2020/03/27

### Summary

Kong Enterprise k8s 2.0.2.0 is based on Kong Gateway 2.0.2.

### Kong Gateway

Kong Enterprise k8s 2.0.2.0 is based on Kong Gateway 2.0.2,
meaning it inherits all new features and changes between Kong Gateway 1.4.2 and
2.0.2.
Please review the changelog for
[Kong Gateway](https://github.com/Kong/kong/blob/master/CHANGELOG.md).

> ### Plugins

Reviewer: ```suggestion
### Enterprise Plugins
```
Basing this on the section title for 1.3.0.0

> Follow new enterprise-only plugins have been added in this release:

Reviewer: ```suggestion
The following plugins are included in this release:
```
Basing on entry for 1.3.0.0
